### DISCUSSION TITLE
[Help] Changing Color Scheme in Interactive Mode

### DISCUSSION THREAD

**Discussion by niceiq** [zk-org/zk#345] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

![1694107443](https://github.com/mickael-menu/zk/assets/98485752/056d6e4c-b361-44d0-948a-cca8dd5255fe)

Hi, I find myself using the interactive mode quite a bit, and when I recently switched to a light terminal theme, everything kind of went ugly.

Do I need to change things in the configs? Or is it something I need to do on my terminal config?

I use Alacritty as a terminal emulator.

Thank you!

**Comment by mickael-menu** [comment, marked as answer]
If you want to customize the left side, you can change the [`fzf-line` config](https://github.com/mickael-menu/zk/blob/main/docs/tool-fzf.md#line-format) paired with a [`{{style}}` template helper](https://github.com/mickael-menu/zk/blob/main/docs/style.md) to customize the colors.

The right side (file preview) is outside the control of `zk`. You need to configure whichever [`fzf-preview` tool you use](https://github.com/mickael-menu/zk/blob/main/docs/tool-fzf.md#preview-command) (assuming `bat` from your screenshot).

**Comment by niceiq** [comment]
Thanks a lot! 

Super helpful, thank you for writing such an amazing program I use it every day!
